db/models/getArticles.model.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../connection");
+
+exports.findArticles = () => {
+    return db.query(`SELECT article_id, title, topic, author, created_at, votes, article_img_url, COUNT(body) AS comment_count FROM articles 
```

Comment:
You won't be counting a body as the comment count. The comment count is meant to reflect how many comments have the same article id as the article in your list. You'll need to make a join.